### ISSUE TITLE
[Suggestion] Put color selections in the config.json file

### ISSUE THREAD

**Issue by hickabilly** [PocketMobsters/PokeMobBot#575]
Some of the default color selections are difficult to read in the console if it is launched in cmd.exe (black background) versus in PowerShell (blue background).  Consider moving the color definitions to the config.json file so that they are not static.

**Comment by jinkawasaki333** [comment]
It's already colorful. Are you not using the release version?

**Comment by Bananab0y** [comment]
I agree with you. Would be a nice little feature.

@jinkawasaki333 I think you misunderstood him. He is asking for a setting for custom colors because some of them are difficult to read when the bot is being used in cmd.

**Comment by hickabilly** [comment]
Yes, colors are used for the display, but they are statically coded.  If you use the default cmd.exe to run the Bot, the darker colors are difficult to see.  If you run the application in PowerShell, the blue text is unreadable.  This is asking to move the colors from static bindings in the code to configurable items in the .json files so that the end user can configure the colors for the display depending on how they have their system set up.

**Comment by Austeri** [comment]
I'd be interested in seeing this feature

**Comment by lucabravi** [comment]
Really interested in it
I really hate color of eggs console log

**Comment by nerclid** [comment]
I would like this feature also.
